components/tools/OmeroPy/src/omero/plugins/prefs.py

```diff
@@ -150,7 +150,12 @@ def _configure(self, parser):
             "KEY", nargs="*", help="Names of keys in the current profile")
         get.add_argument(
             "--hide-password", action="store_true",
-            help="Hide values of password keys in the current profile")
+            help="[Deprecated; this is the default now] Hide values of "
+                 "password keys in the current profile")
+        get.add_argument(
+            "--show-secrets", action="store_true",
```

Comment:
You can also have `--show-secrets` store into the same variable (`hide_secrets`) by using `store_false` with the `target` argument. This would keep the rest of the logic the same. See https://github.com/openmicroscopy/openmicroscopy/blob/v5.4.0/components/tools/OmeroPy/src/omero/plugins/metadata.py#L171

That example also uses `add_mutually_exclusive_group()` which may be of interested.